docs/src/components/GestureExamples/PanExample/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { StyleSheet } from 'react-native';
+import Animated, {
+  clamp,
+  useAnimatedStyle,
+  useSharedValue,
+} from 'react-native-reanimated';
+import {
+  Gesture,
+  GestureDetector,
+  GestureHandlerRootView,
+} from 'react-native-gesture-handler';
+import Hand from '@site/static/img/hand-two.svg';
+import stylesWeb from './styles.module.css';
+import { RADIUS, isInsideCircle, useStylesForExample } from '../utils';
+
+export default function DecayBasicExample() {
+  const colorModeStyles = useStylesForExample();
+  const [showHand, setShowHand] = useState(true);
+  const offsetX = useSharedValue(0);
+  const offsetY = useSharedValue(0);
+
+  const pan = Gesture.Pan()
+    .onStart(() => setShowHand(false))
+    .onChange((event) => {
+      offsetX.value =
+        Math.abs(offsetX.value) <= 100
+          ? offsetX.value + event.changeX >= 100
+            ? 100
+            : offsetX.value + event.changeX <= -100
+            ? -100
+            : offsetX.value + event.changeX
+          : offsetX.value;
+      offsetY.value =
+        Math.abs(offsetY.value) <= 100
+          ? offsetY.value + event.changeY >= 100
+            ? 100
+            : offsetY.value + event.changeY <= -100
+            ? -100
+            : offsetY.value + event.changeY
+          : offsetY.value;
+    })
+    .onEnd(() => setShowHand(true));
```

Comment:
```suggestion
    .onEnd(() => setShowHand(true))
    .minDistance(0);
```
add this here to make it more responsive